governance/processes.md

```diff
@@ -168,10 +207,3 @@ issue, but must recuse themselves from voting on the issue.
 Core contributors can also voluntarily leave the project by notifying the community through a public means or by notifying the entire council.
 
 Unless they request otherwise, they will be listed on {ref}`emeritus` page.
-
-## Voting Criteria For Future Elections
-Voting for first election is restricted to establish stable governance, and to defer major decision to elected leaders
-* For the first election only the people registered following the guidelines in `elections/ArviZ_2020.md` can vote
-* In the first year, the council must determine voting eligibility for future elections between two criteria:
-  * Core contributors
-  * The contributing community at large
```

Comment:
I have deleted this part, however, we might want to reconsider (or consider as we didn't really discuss this when adding the recurring contributor figure). Do we want recurrent contributors to vote on the election? That would mean they can't be candidates nor have commit rights but they would be able to vote in the elections in addition to being in slack